db/seeds.rb

```diff
@@ -5,3 +5,237 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# Pdfs
+Pdf.create(url: "public/LearnMore03.pdf", page: "tools and resources", slot: 1)
+
+# Cities
+Company.create(
+  name: "company",
+  organization: "org",
+  title: "title",
+  state: "Kentucky",
+  city: "Louisville",
+  email: "email@email.com",
+  hiring: true,
+  hire_count: 100
+)
+
+Company.create(
+  name: "company",
+  organization: "org",
+  title: "title",
+  state: "state",
+  city: "New York City",
+  email: "email@email.com",
+  hiring: true,
+  hire_count: 100
+  )
+
+Company.create(
+  name: "company",
+  organization: "org",
+  title: "title",
+  state: "state",
+  city: "Philadelphia",
+  email: "email@email.com",
+  hiring: true,
+  hire_count: 100
+  )
+
+Company.create(
+  name: "company",
+  organization: "org",
+  title: "title",
+  state: "state",
+  city: "City of Kearney and Buffalo County",
+  email: "email@email.com",
+  hiring: true,
+  hire_count: 100
+  )
+
+Company.create(
```

Comment:
This is all the prelim companies that have signed up. I needed seeds to try to get marks rendered on the map.